public/content/sv/welcome/install-d-locally.md

```diff
@@ -0,0 +1,43 @@
+# Installera D lokalt
+
+På D språkets webbsida [dlang.org](https://dlang.org) kan den senaste kompilerar
+versionen av referens kompilatorn **DMD** (Digital Mars D)
+[laddas ned](http://dlang.org/download.html) och installeras:
+
+### Windows
+
+* [Installerare](http://downloads.dlang.org/releases/2.x/2.071.1/dmd-2.071.1.exe)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.windows.7z)
+* genom [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
+
+* `.dmg` [fil](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.dmg)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.osx.tar.xz)
+* genom [Homebrew](http://brew.sh): `brew install dmd`
+
+### Linux / FreeBSD
+
+För att snabbt installera dmd i användar katalogen, kör `curl -fsS
```

Comment:
Spelling mistake `användar katalogen` -> `användarkatalogen`
But I think it would be better as `användarens hemkatalog` or `användarens hemmapp`